Form/EventListener/ResizeFormListener.php

```diff
@@ -63,8 +63,8 @@ public static function getSubscribedEvents()
     {
         return array(
             FormEvents::PRE_SET_DATA    => 'preSetData',
-            FormEvents::PRE_BIND        => 'preBind',
-            FormEvents::BIND            => 'onBind',
+            FormEvents::PRE_SUBMIT      => 'preSubmit',
```

Comment:
I think I fix those code ... also you need to keep the original method name to avoid BC break